apps/api-extractor/src/generators/ApiReportGenerator.ts

```diff
@@ -426,15 +496,50 @@ export class ApiReportGenerator {
           }
         }
 
-        ApiReportGenerator._modifySpan(collector, child, entity, childAstDeclaration, insideTypeLiteral);
+        ApiReportGenerator._modifySpan(
+          collector,
+          child,
+          entity,
+          childAstDeclaration,
+          insideTypeLiteral,
+          releaseLevel
+        );
       }
     }
   }
 
-  private static _shouldIncludeInReport(astDeclaration: AstDeclaration): boolean {
+  private static _shouldIncludeInReport(
+    collector: Collector,
+    astDeclaration: AstDeclaration,
+    releaseLevel: ApiReportReleaseLevel
+  ): boolean {
     // Private declarations are not included in the API report
     // eslint-disable-next-line no-bitwise
-    return (astDeclaration.modifierFlags & ts.ModifierFlags.Private) === 0;
+    if ((astDeclaration.modifierFlags & ts.ModifierFlags.Private) !== 0) {
+      return false;
+    }
+
+    const apiItemMetadata: ApiItemMetadata = collector.fetchApiItemMetadata(astDeclaration);
+
+    // No specified release tag is considered the same as `@public`.
+    const releaseTag: ReleaseTag =
+      apiItemMetadata.effectiveReleaseTag === ReleaseTag.None
+        ? ReleaseTag.Public
+        : apiItemMetadata.effectiveReleaseTag;
+
+    // If the declaration has a release tag that is not in scope, omit it from the report.
+    switch (releaseLevel) {
+      case ApiReportReleaseLevel.Untrimmed:
```

Comment:
The term "trimming" has always referred to .d.ts rollups.  It may be confusing to introduce this term for .api.md files, because even though it involves deleting unnecessary declarations, the motivation and details are rather different in this context.

Compare with an alternative possible design like this:
```js
  // the base filename
  "reportFileName": "<unscopedPackageName>.api.md",
  
   // magically transforms to produce my-app.alpha.api.md and my-app.beta+public.api.md
  "reportVariations": [ "alpha", "beta+public" ]
```

Reply:
I think the "trimming" terminology may make sense if there is intentional parallelism between the reports at a given release level and the rollup at the same level. But I'm not tied to it. 

I do like the simpler config model, though. I'll proceed with that change while we continue to iterate on report structure / semantics.